ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/deduplication/CommandDeduplicationBase.scala

```diff
@@ -330,6 +332,33 @@ private[testtool] abstract class CommandDeduplicationBase(
     }
   })
 
+  test(
+    s"${testNamingPrefix}DeduplicateParallelSubmissions",
+    "Commands submitted at the same, in parallel, should be deduplicated",
+    allocate(SingleParty),
+  )(implicit ec => { case Participants(Participant(ledger, party)) =>
+    lazy val request = ledger
+      .submitRequest(party, DummyWithAnnotation(party, "Duplicate").create.command)
+      .update(
+        _.commands.deduplicationPeriod := DeduplicationPeriod.DeduplicationTime(
+          deduplicationDuration.asProtobuf
+        )
+      )
+    val numberOfParallelRequests = 10
+    Future
+      .traverse(Seq.fill(numberOfParallelRequests)(request))(request => {
+        submitRequestAndGetStatusCode(ledger)(request, party)
+      })
```

Comment:
Shouldn't they have different submission IDs?

Reply:
Ah I see, you are relying on autogeneration, right?

Reply:
If we don't set the submission id then one is generated, which is always different.